README.md

```diff
@@ -52,7 +52,7 @@ Here are flows to try out, from simple to specific detailed variants.
 
 - **[Simple flow](READMEs/datatokens-flow.md)** - the essence of Ocean - creating a data NFT & datatoken.
```

Comment:
You need to link to the new consume-flow.md file. I recommend to do it right after datatokens-flow.md